src/clippy.rs

```diff
@@ -217,6 +227,24 @@ mod tests {
             no_default_features_command_parameters,
             no_default_features_linter.get_command_parameters()
         );
+
+        let all_features_linter = linter
```

Comment:
There's even a test for the new flag, it's wonderful ! 🎉

Reply:
YAY! Putting it in Ready for Review. Thanks!